### PR TITLE
NUI-5934: Fix Percy tests with random data

### PR DESCRIPTION
**Pull request by NataliiaPodpriatova**

### CODE REVIEW

File: packages/bits/demo/src/components/demo/common/tree/tree-visual-test/data.ts [hunk 1]
```diff
@@ -58,8 +60,12 @@ export class HttpMock {
         return of(response).pipe(delay(1000));
     }
 
-    private getRandomFrom(list: any[]) {
-        return list[Math.floor(Math.random() * list.length)];
+    private getTotalItemList(list: any[], times: number) {
+        const totalArray = [];
+        for(let i = 0; i < times; i++) {
```
Comment (Vita-Karas) [inline packages/bits/demo/src/components/demo/common/tree/tree-visual-test/data.ts:65]: missing indent after `for`

File: packages/bits/demo/src/components/demo/convenience/time-frame-bar/time-frame-bar-visual/time-frame-bar-visual.component.ts [hunk 2]
```diff
@@ -13,12 +16,15 @@ class TestBar {
 
     public zoomIn = (value: ITimeframe) => this.timeFrame = this.history.save(value);
     public undo = () => this.timeFrame = this.history.undo();
-    public change = (value?: ITimeframe) => this.timeFrame = this.history.restart(value);
+    public restart = (value?: ITimeframe) => this.timeFrame = this.history.restart(value);
+    public change = (value: ITimeframe) => {
```
Comment (Vita-Karas) [inline packages/bits/demo/src/components/demo/convenience/time-frame-bar/time-frame-bar-visual/time-frame-bar-visual.component.ts:20]: parentheses can be removed

**Review by dmytrofirsov — APPROVED** [review]

File: packages/bits/demo/src/components/demo/common/tree/tree-visual-test/data.ts (hunk 1 above)
Comment (NataliiaPodpriatova) [reply packages/bits/demo/src/components/demo/common/tree/tree-visual-test/data.ts]: I'll fix it, thank you

File: packages/bits/demo/src/components/demo/convenience/time-frame-bar/time-frame-bar-visual/time-frame-bar-visual.component.ts (hunk 2 above)
Comment (NataliiaPodpriatova) [reply packages/bits/demo/src/components/demo/convenience/time-frame-bar/time-frame-bar-visual/time-frame-bar-visual.component.ts]: I'll fix it, thank you

**Review by dmytrofirsov — APPROVED** [review]

File: packages/bits/demo/src/components/demo/convenience/time-frame-bar/time-frame-bar-visual/time-frame-bar-visual.component.ts (hunk 2 above)
Comment (Vita-Karas) [reply packages/bits/demo/src/components/demo/convenience/time-frame-bar/time-frame-bar-visual/time-frame-bar-visual.component.ts]: These methods are similar, please fix

**Review by Vita-Karas — APPROVED** [review]

**Review by dmytrofirsov — APPROVED** [review]

**Pull request merged**